### PR TITLE
Feat: Added Rating Component

### PR DESCRIPTION
**Pull request by shuntzhere**

### CODE REVIEW

File: components/rating/rating.html [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/55aa2069c0.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="/css/base.css" />
+    <title>Rating</title>
```
Comment (kjais1720) [inline components/rating/rating.html]: Add your CL name here as well

File: components/rating/rating.html [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  </head>
+  <body>
+    <div class="container">
+      <div class="nav-container">
+        <nav>
+          <div class="left-nav flex flex-items-center justify-between">
+            <img class="hamburger" src="/assets/svg/hamburger-solid.svg" />
+            <a href="../../../index.html">
```
Comment (kjais1720) [inline components/rating/rating.html]: You can navigate to the root folder directly with a single slash `/`

File: components/rating/rating.html [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+              <img class="logo flex-grow" src="/assets/svg/logo.svg" />
```
Comment (kjais1720) [inline components/rating/rating.html:21]: You can specify the value for the `flex-grow` property in the classname as well

**Review by kjais1720 — APPROVED** [review]
Looks good overall, just a couple of suggestions as mentioned above

**Review by sadathshariff — APPROVED** [review]
looks good, go ahead and merge

File: components/modal/modal.html [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+            <a href="../../../index.html">
+              <img class="logo flex-grow" src="/assets/svg/logo.svg" />
+            </a>
+            <a
+              class="right-nav flex"
+              href="https://github.com/shuntzhere/Shore-UI"
+              >Github</a
+            >
+          </div>
+          <!-- <div></div> -->
```
Comment (tanishq20) [inline components/modal/modal.html:25]: remove this
(Apply this in whole code)

**Review by tanishq20 — APPROVED** [review]
Everything looks good.

**Pull request merged**